avm/res/web/serverfarm/tests/e2e/max/main.test.bicep

```
@@ -0,0 +1,107 @@
+targetScope = 'subscription'
+
+metadata name = 'Using max parameter set'
+metadata description = 'This instance deploys the module with a full set of parameters for a single scenario.'
```

Usually we try to keep it generic as all this test is supposed to do is try and throw as many parameters at the template as possible. It's not per se a 'scenario' but more just a technical test.
```suggestion
metadata name = 'Using large parameter set'
metadata description = 'This instance deploys the module with most of its features enabled.'
```